Annotations/AddCustomStampInCustomizedButton/AddCustomStampInCustomizedButton_NET.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.10.35122.118
```

Comment:
Change the VS version of the solution file in all projects. Refer the solution file of our source

Reply:
Check the VS code version and changes completed